test_scripts/Polices/Policy_Table_Update/003_ATF_P_TC_PTU_NotSuccessful_AppID_ListedPT_NewIgnCycle.lua

```diff
@@ -0,0 +1,182 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- [PolicyTableUpdate] Policy Table Update in case of failed retry strategy during previour IGN_ON (SDL.PolicyUpdate)
+-- [HMI API] PolicyUpdate request/response
+--
+-- Description:
+-- SDL should request in case of failed retry strategy during previour IGN_ON
+-- 1. Used preconditions
+-- SDL is built with "-DEXTENDED_POLICY: EXTERNAL_PROPRIETARY" flag
```

Comment:
@maria-mih, according to specification is DEXTENDED, in real usage of Cmake is EXTENDED